build/Dockerfile

```diff
@@ -1,4 +1,14 @@
-FROM golang:1.13.7-alpine3.11 as builder
+FROM registry.access.redhat.com/devtools/go-toolset-rhel7:1.12.12-4 as builder
```

Comment:
If you use go 1.13, you can do this:

```
# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhel8/go-toolset
FROM rhel8/go-toolset:1.13.4-15 as builder
ENV GOPATH=/go/
```

Reply:
```
Error response from daemon: pull access denied for rhel8/go-toolset, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

Do we need to start implementing/maintaining `Dockerfile` and `rhel.Dockerfile`?

Reply:
@nickboldt now `registry.redhat.io/rhel8/go-toolset:1.13.4-15` is used in rhel.Dockerfile.